src/frontend/src/flows/manage/index.ts

```diff
@@ -222,52 +222,58 @@ export const displayManage = (
         "More than one recovery keys are registered, which is unexpected. Only one will be shown."
       );
     }
-    displayManagePage({
-      userNumber,
-      devices,
-      onAddDevice: async () => {
-        const nextAction = await chooseDeviceAddFlow();
-        switch (nextAction) {
-          case "canceled": {
-            resolve();
-            break;
-          }
-          case "local": {
-            await addLocalDevice(userNumber, connection, devices_);
-            resolve();
-            break;
+    const display = () =>
+      displayManagePage({
+        userNumber,
+        devices,
+        onAddDevice: async () => {
+          const nextAction = await chooseDeviceAddFlow();
+          switch (nextAction) {
+            case "canceled": {
+              resolve();
```

Comment:
You can avoid `resolve` duplication:

```
switch (nextAction) {
           case "cancel":
                 break;
            case "local": {
              await addLocalDevice(userNumber, connection, devices_);
              break;
            }
            case "remote": {
              await addRemoteDevice({ userNumber, connection });
              break;
            }
            default:
              unreachable(nextAction);
          }
          
          resolve();
     }
```

Reply:
I think this is just formatted code, see for commit for actual changes

Reply:
Oki doki, might reformat it some day...or not ;)